tests/test_welcome.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from dreamhome import app
+
+class TestWelcome(unittest.TestCase):
+    def setUp(self):
+        self.app = app.test_client()
+
+    def test_landing_page(self):
+        # When
+        response = self.app.get('/')
+
+        # Then
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data, b'Welcome to your dream home server!')
```

Comment:
Testing syntax looks pretty similar to our unit tests.

Looks like a pretty standard unit test

Reply:
Awesome! We are using `unittest`, a native for python